sdk/storage/azure-storage-file-datalake/azure/storage/filedatalake/_file_system_client.py

```diff
@@ -753,11 +756,6 @@ def create_file(self, file,  # type: Union[FileProperties, str]
             The file with which to interact. This can either be the name of the file,
             or an instance of FileProperties.
         :type file: str or ~azure.storage.filedatalake.FileProperties
-        :param ~azure.storage.filedatalake.ContentSettings content_settings:
-            ContentSettings object used to set path properties.
-        :param metadata:
-            Name-value pairs associated with the file as metadata.
-        :type metadata: dict(str, str)
```

Comment:
I assume these were meant to be `keyword`. We shouldn't remove them but change them to `keyword`/`paramtype`.